src/lib/anonymizer-service/index.js

```diff
@@ -133,7 +133,7 @@ const getPhonyValuesByCol = (hashMap, piiCols, mysqlHelper) => {
 
 const getPiiInsertQueries = (hashes, lookupTable, phonyTable) => {
     const insertPhonyLookupQuery = [
-        `INSERT INTO ${lookupTable} (PIIHashValue, PhonyValue)`,
+        `INSERT IGNORE INTO ${lookupTable} (PIIHashValue, PhonyValue)`,
```

Comment:
My gut is telling me this could result in swallowing of potentially significant errors.

Reply:
The reason I included the 'IGNORE' option in the sql statement is because when sending a query with multiple statements to insert items into a lookup table, mysql will throw an error and halt any remaining insert statements if one of these inserts attempts to add a pii-hash that already exists in the lookup table (this can happen if two parallel processes are trying to add the same pii-hash to a lookup table). Ideally, this scenario should simply be ignored without throwing an error and having to retry the transaction. I attempt to catch other errors that get swallowed by reading the responses. If an insert statement returns a response message that has warnings and the affected rows is set to 0, we know that the insert was not successful. But I agree, we do not know the exact cause for the error so the error gets swallowed.. Looking for a way around this..